File: basic-algorithms/js/01-reverse-string.js
function reverseString(str, arr) {
  arr = str.split(''); str = ''; bnoden = arr.length-1;
  for (var i in arr) str+=arr[bnoden]; bnoden--;
  return str
}
// Test
reverseString("hello") // "olleh"
reverseString("Howdy") // "ydwoH"
reverseString("Greetings from Earth") // "htraE morf sgniteerG"

File: basic-algorithms/01reversestring.go
// https://play.golang.org/p/apC1p7ubf3
package main

import (
	"fmt"
)

func reverseString(str string) string {
	arr, bnoden := []rune(str), []rune{}

	for i := len(arr)-1; i >= 0; i-- {
		bnoden = append(bnoden, arr[i])
	}
	return string(bnoden)
}

// Test
func main() {
	fmt.Println(reverseString("hello"))                // "olleh"
	fmt.Println(reverseString("Howdy"))                // "ydwoH"
	fmt.Println(reverseString("Greetings from Earth")) // "htraE morf sgniteerG"

}
